app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -77,15 +77,15 @@ function buildArticleHTML(article) {
     }
     var saveButton = '';
     if (article.class_name === "Article") {
-      var saveButton = '<button class="article-engagement-count bookmark-engage" data-reactable-id="'+article.id+'">\
-                        <span class="bm-success"></span>\
-                        <span class="bm-initial"></span>\
-                      </button>'
+      saveButton = '<button class="article-engagement-count bookmark-engage" data-reactable-id="'+article.id+'">\
+                      <span class="bm-initial">SAVE</span>\
+                      <span class="bm-success">SAVED</span>\
+                    </button>'
     } else if (article.class_name === "User") {
-      var saveButton = '<button style="width: 122px" class="article-engagement-count bookmark-engage follow-action-button"\
+      saveButton = '<button style="width: 122px" class="article-engagement-count bookmark-engage follow-action-button"\
```

Comment:
I know the style was inline before, but it'd make sense to add it to one of the classes on the button.

Reply:
I feel this part is external to my PR, mostly because I'm not sure what the save button for the `User` class does :)